docs/navigation.md

```diff
@@ -23,11 +23,12 @@ The Operations section lists all the operations that are a part of the AsyncAPI.
 
 ## Messages
 
-This Messages section displays the title of each message used to communicate to the API.
+This Messages section displays the messages stored in `components.messages` section.
```

Comment:
Hey @magicmatatjahu 

I have updated the description for `Messages` and `Schemas`.

Do you want me to add the following line to the description as well?

> - we can have it defined but not used by references in the operation fields

Reply:
Yep, you should add this note, that messages and schemas are defined in components but it can (but not must) be used :)